x-pack/plugins/uptime/server/lib/adapters/monitors/elasticsearch_monitors_adapter.ts

```diff
@@ -261,6 +263,7 @@ export class ElasticsearchMonitorsAdapter implements UMMonitorsAdapter {
                   },
                 },
               ],
+              size: 50,
```

Comment:
This will cause the monitor list to fill with `50` items instead of the defaulted `10`, correct?

Reply:
Yes! The UI lets you select 10-50, but so if you select 10 this would query more than necessary. However, perf-wise it's negligible and easier to hard code for now. We can fix later.

Reply:
++ - totally agree

Reply:
++ totally agree